pkg/controller/admissionchecks/provisioning/controller_test.go

```diff
@@ -642,3 +680,11 @@ func TestReconcile(t *testing.T) {
 	}
 
 }
+
+func setDuringTest(val *int32, newVal int32) func() {
```

Comment:
maybe should be a test util, using generics.

Reply:
done